components/raftstore-v2/src/batch/store.rs

```diff
@@ -375,7 +375,7 @@ impl<EK: KvEngine, ER: RaftEngine> StoreSystem<EK, ER> {
             .store_writers
             .spawn(store_id, raft_engine.clone(), None, router, &trans, &cfg)?;
 
-        let snap_mgr = SnapManagerBuilder::default().build(
+        let snap_mgr = TabletSnapManager::new(
             tablet_factory
                 .tablets_path()
                 .join(Path::new("snap"))
```

Comment:
In snapshot-v1，the snapshot path is: data_dir/snap. 
And in past , the snapshot root path is : data_dir/tablets/snap.
I don't know this is belongs to the expect.